client/client.go

```diff
@@ -324,6 +324,8 @@ const (
 	ErrorKindSnapAlreadyInstalled = "snap-already-installed"
 	ErrorKindSnapNotInstalled     = "snap-not-installed"
 	ErrorKindNoUpdateAvailable    = "snap-no-update-available"
+
+	ErrorKindNotSnap = "snap-not-snap"
```

Comment:
Indeed, I think that clearer. I fixed the tag. I'm a bit on the fence about the kind, i.e. `ErrorKindNotASnap` reads a bit strange (but maybe that is just me).